<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QWidget#widget{ background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0.0115909, y2:0.034, stop:0 rgba(177, 203, 228, 255), stop:0.806818 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255))}

</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 28pt &quot;Calibri Light&quot;;</string>
    </property>
    <property name="text">
     <string>Animal Shelter </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 28pt &quot;Calibri Light&quot;;</string>
    </property>
    <property name="text">
     <string> Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadB">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);

font: 25 16pt &quot;Microsoft YaHei UI Light&quot;;
border: 1px solid;

</string>
    </property>
    <property name="text">
     <string>Read Info</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>50</y>
      <width>41</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="WriteB">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);

font: 25 16pt &quot;Microsoft YaHei UI Light&quot;;
border: 1px solid;

</string>
    </property>
    <property name="text">
     <string>Add Info</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QPushButton" name="EditB">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);

font: 25 16pt &quot;Microsoft YaHei UI Light&quot;;
border: 1px solid;

</string>
    </property>
    <property name="text">
     <string>Edit Info</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>581</y>
      <width>801</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>16</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
